### PR TITLE
Added Statistics to Admin Page

### PR DESCRIPTION
**Pull request by CarsonFiechtner**
Added statistics as well as made password black dots at login

### CODE REVIEW

**Comment by Leksyk** [comment]
To clarify - only the comment about date formatting is required here. The rest can be applied in a separate pull request.

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 1]
```diff
@@ -84,6 +89,52 @@ public User getUser(String username) {
     return null;
   }
 
+  /**
+   * Access the newest User object.
+   *
+   * @return The newest User.
+   */
+  public String getNewestUser() {
```
[the remark this answers is not in the document]
Comment (andrewjamesphillips) [reply src/main/java/codeu/model/store/basic/UserStore.java]: Each time a new user is added we could update newName.  That way we would never need to iterate over all the users, instead we would always have the updated name in O(1) time

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 2]
```diff
@@ -84,6 +89,52 @@ public User getUser(String username) {
...
+    }
+    return newName;
+  }
+
+  /**
+   * Access the oldest User object.
+   *
+   * @return The oldest User.
+   */
+  public String getOldestUser() {
```
[the remark this answers is not in the document]
Comment (andrewjamesphillips) [reply src/main/java/codeu/model/store/basic/UserStore.java]: Same here: IF the oldest user is deleted, we could check once to find the next oldest user and store it in oldName, that we don't need to check through all users each time

File: src/main/webapp/WEB-INF/view/login.jsp [hunk 3]
```diff
@@ -45,7 +45,7 @@
       <input type="text" name="username" id="username">
       <br>
       <Label for="password">Password: </label>
```
Comment (andrewjamesphillips) [inline src/main/webapp/WEB-INF/view/login.jsp]: Good catch

File: src/main/webapp/WEB-INF/view/login.jsp [hunk 4]
```diff
@@ -45,7 +45,7 @@
...
-      <input type="text" name="password" id="password">
+      <input type="password" name="password" id="password">
```
Comment (andrewjamesphillips) [inline src/main/webapp/WEB-INF/view/login.jsp]: Good catch

**Pull request closed without merging**